debug_toolbar/settings.py

```diff
@@ -41,6 +41,7 @@
     "SHOW_TEMPLATE_CONTEXT": True,
     "SKIP_TEMPLATE_PREFIXES": ("django/forms/widgets/", "admin/widgets/"),
     "SQL_WARNING_THRESHOLD": 500,  # milliseconds
+    "SQL_TOKEN_GROUPING": True,
```

Comment:
To keep in line with the rest of the settings, this should likely be either `ENABLE_SQL_TOKEN_GROUPING` or `SQL_ENABLE_TOKEN_GROUPING`. @matthiask opinions?

Reply:
The pull request which introduced the code in question was this one: https://github.com/jazzband/django-debug-toolbar/pull/1116

I don't think anyone understands what "token grouping" should be in this context. Maybe something like `PRETTIFY_SQL` would describe the intent better and allow us to disable or enable additional prettification features with one setting. The `NOUN_VERB` format seems to be used with many settings, e.g. "show template context", "enable stacktraces" etc.